test/Altinn.Platform.Events.Tests/TestingServices/RegisterServiceTest.cs

```diff
@@ -76,6 +81,49 @@ public async Task PartyLookup_MatchFound_IdReturned()
             _handlerMock.VerifyAll();
         }
 
+        [Fact]
+        public async Task GetParty_SuccessResponse_PartyTypeDeserializedSuccessfully()
+        {
+            // Arrange         
+            PartyType expectedPartyType = PartyType.Organisation;
+
+            string repsonseString = "{\"partyId\": 500000," +
+                "\"partyTypeName\": \"Organisation\"," +
+                "\"orgNumber\": \"897069650\"," +
+                "\"unitType\": \"AS\"," +
+                "\"name\": \"DDG Fitness\"," +
+                "\"isDeleted\": false," +
+                "\"onlyHierarchyElementWithNoAccess\": false," +
+                "\"organization\": {\"orgNumber\": \"897069650\",\"name\": \"DDG Fitness\",\"unitType\": \"AS\",\"telephoneNumber\": \"12345678\",\"mobileNumber\": \"92010000\",\"faxNumber\": \"92110000\",\"eMailAddress\": \"central@ddgfitness.no\",\"internetAddress\": \"http://ddgfitness.no\",\"mailingAddress\": \"Sofies Gate 1\",\"mailingPostalCode\": \"0170\",\"mailingPostalCity\": \"Oslo\",\"businessAddress\": \"Sofies Gate 1\",\"businessPostalCode\": \"0170\",\"businessPostalCity\": \"By\",\"unitStatus\": null},\"childParties\": null\r\n}";
+
+            HttpResponseMessage httpResponseMessage = new()
+            {
+                StatusCode = HttpStatusCode.OK,
+                Content = new StringContent(repsonseString, Encoding.UTF8, "application/json")
+            };
+
+            HttpRequestMessage actualRequest = null;
+            void SetRequest(HttpRequestMessage request) => actualRequest = request;
+            InitializeMocks(httpResponseMessage, SetRequest);
+
+            HttpClient httpClient = new HttpClient(_handlerMock.Object);
+
+            RegisterService target = new RegisterService(
+                httpClient,
+                _contextAccessor.Object,
+                _accessTokenGenerator.Object,
+                _generalSettings.Object,
+                _platformSettings.Object,
+                new Mock<ILogger<RegisterService>>().Object);
+
+            // Act
+            Party actual = await target.GetParty(500000);
+
+            // Assert
+            Assert.Equal(expectedPartyType, actual.PartyTypeName);
+            _handlerMock.VerifyAll();
```

Comment:
Not a new issue unique to this test, but I'm honestly not sure what this statement does. What is there to verify?

Reply:
Simply copied from the previous tests. Didn't seem to have an effect when I comment out the part of the code that sends the request. If verifyAll doesn't verify that the method set up is called, we can simply remove it. Removing for all the tests

Reply:
Thank you.